c++/src/kj/configure.bzl

```diff
@@ -78,7 +78,8 @@ def kj_configure():
         name = "kj-defines",
         defines = [
             # todo: take this from flag
-            "KJ_CONTENTION_WARNING_THRESHOLD=100",
+            # todo: define below creates failure on macOS
+            # "KJ_CONTENTION_WARNING_THRESHOLD=100",
```

Comment:
We should just remove this. This is an option that dependents should be setting in `.bazelrc` if they want it.